lib/game.rb

```diff
@@ -1,7 +1,7 @@
 class Game
```

Comment:
in our final refactor: you can inherit as Darby and i do and just initialize the additional data set needed